Readme.md

```diff
@@ -1,26 +1,23 @@
+This example shows how to change a default or user-defined [dashboard parameter](https://docs.devexpress.com/Dashboard/400012) value in the [`DashboardControl.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters) event handler. The effective parameter value is hidden from users. Set the [`Visible`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardCommon.DashboardParameter.Visible) property to `false` to hide the parameter from the **Dashboard Parameters** dialog.
 
-To accomplish this task, a parameter named <strong>parameterState</strong> is added to the dashboard. It has a default value and a list of values to show in a look-up editor. A <a href="https://docs.devexpress.com/Dashboard/400012/building-the-designer-and-viewer-applications/wpf-viewer/manage-dashboard-parameters">Dashboard Parameters dialog</a> displays the values and allows the end-user to select a parameter value in the list.<br>However, by handling the <a href="https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters">DashboardControl.CustomParameters</a> event, we can validate the parameter value and ignore the value provided by the end-user. To accomplish this, source data is filtered using a <strong>parameterState </strong>parameter.The value of this parameter is changed at runtime by handling the <a href="https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters">DashboardControl.CustomParameters</a> event which is raised before the dashboard sends a query to a database. Thus, only the value passed in the <a href="https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters">DashboardControl.CustomParameters</a> event is in effect.<br>
+In this example, the **parameterState** dashboard parameter whit static look-up settings is added to the dashboard. This dashboard parameter is used to filter the data source. The [Dashboard Parameters](https://docs.devexpress.com/Dashboard/400012) displays the values and enables users to select a parameter value from the list. In the [`DashboardControl.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters) event handler, we can change the value provided by the user before it is passed to the query.
```

Comment:
1. Please change "whit static look-up settings" to "**with** static look-up settings".

2. `The [Dashboard Parameters](https://docs.devexpress.com/Dashboard/400012) displays the values and enables users to select a parameter value from the list.`

Do you mean "The Dashboard Parameters display" (plural) or "The Dashboard Parameter displays" (singular)?

3. `In the [`DashboardControl.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters) event handler, we can change the value provided by the user before it is passed to the query.`

Please change "provided by" to "supplied by".

Reply:
```suggestion
In this example, the **parameterState** dashboard parameter with static look-up settings is added to the dashboard. This dashboard parameter is used to filter the data source. The [Dashboard Parameters](https://docs.devexpress.com/Dashboard/400012) dialog displays the values and enables users to select a parameter value from the list. In the [`DashboardControl.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWpf.DashboardControl.CustomParameters) event handler, we can change the value supplied by the user before it is passed to the query.
```